src/Type/JsonDocumentType.php

```diff
@@ -137,6 +137,14 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
         return $this->getSerializer()->deserialize($value, '', $this->format, $this->deserializationContext);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getSqlDeclaration(array $fieldDeclaration, AbstractPlatform $platform)
+    {
+        return sprintf('JSON');
```

Comment:
`sprintf` is useless here and should be removed.

Reply:
Yeah, I was following the pattern I saw somewhere else, but I agree.